Package/gulpfile.js

```diff
@@ -8,13 +8,36 @@ const shelljs = require('shelljs');
 const rename = require('gulp-rename');
 const glob = require('glob');
 const chalk = require('chalk');
+const marked = require('marked');
 
 let assemblediOSAndroidDir = 'Assembled-iOSAndroid';
 let assembledWindowsDir = 'Assembled-Windows';
 let basekitBuild = false;
 let cmakeBasekitBuildDefinition = '';
 let basekitPackagePath = '';
 
+
+const makeHTMLNotice = async () => {
```

Comment:
This function is marked async but then uses fs with callbacks. I believe these days you can do `const fs = require('fs/promises')` instead and use Promise returning async functions.